digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/process/ProjectDeltaCalculatorSpec.scala

```diff
@@ -327,6 +326,147 @@ class ProjectDeltaCalculatorSpec extends FunSuite with Matchers {
+        ProjectLink(1080,roadNumber,roadPartNumber,Track.Combined,Discontinuity.Continuous,7743,7924,4102,4283,None,None,Some(createdBy),247460,0.0,180.89,AgainstDigitizing,(NoCP,NoCP),(NoCP,NoCP),List(Point(419761.0,7065398.0,0.0), Point(419739.0,7065578.0,0.0)),projectId,LinkStatus.Transfer,AdministrativeClass.State,FrozenLinkInterface,180.89,31341,265171,12,false,None,1446398762000L,64356,roadName,None,None,None,None,None,None),
+        ProjectLink(1081,roadNumber,roadPartNumber,Track.Combined,Discontinuity.Continuous,7924,7990,4283,4349,None,None,Some(createdBy),247462,0.0,65.605,AgainstDigitizing,(NoCP,NoCP),(NoCP,NoCP),List(Point(419762.0,7065333.0,0.0), Point(419761.0,7065398.0,0.0)),projectId,LinkStatus.Transfer,AdministrativeClass.State,FrozenLinkInterface,65.605,31341,265172,12,false,None,1446398762000L,64356,roadName,None,None,None,None,None,None),
+        ProjectLink(1082,roadNumber,roadPartNumber,Track.Combined,Discontinuity.Continuous,7990,8206,4349,4565,None,None,Some(createdBy),10952140,0.0,215.81,AgainstDigitizing,(NoCP,NoCP),(NoCP,NoCP),List(Point(419852.0,7065147.0,0.0), Point(419762.0,7065333.0,0.0)),projectId,LinkStatus.Transfer,AdministrativeClass.State,FrozenLinkInterface,215.81,31341,265173,12,false,None,1537228663000L,64356,roadName,None,None,None,None,None,None),
+        ProjectLink(1083,roadNumber,roadPartNumber,Track.Combined,Discontinuity.Continuous,8206,8277,4565,4636,None,None,Some(createdBy),248468,0.0,70.731,AgainstDigitizing,(NoCP,NoCP),(NoCP,NoCP),List(Point(419913.0,7065112.0,0.0), Point(419852.0,7065147.0,0.0)),projectId,LinkStatus.Transfer,AdministrativeClass.State,FrozenLinkInterface,70.731,31341,265174,12,false,None,1446398762000L,64356,roadName,None,None,None,None,None,None),
+        ProjectLink(1084,roadNumber,roadPartNumber,Track.Combined,Discontinuity.Continuous,8277,8301,4636,4660,None,None,Some(createdBy),248469,0.0,23.829,AgainstDigitizing,(NoCP,NoCP),(NoCP,NoCP),List(Point(419933.0,7065099.0,0.0), Point(419913.0,7065112.0,0.0)),projectId,LinkStatus.Transfer,AdministrativeClass.State,FrozenLinkInterface,23.829,31341,265175,12,false,None,1446398762000L,64356,roadName,None,None,None,None,None,None),
+        ProjectLink(1085,roadNumber,roadPartNumber,Track.Combined,Discontinuity.Continuous,8301,8469,4660,4828,None,None,Some(createdBy),10949334,0.0,167.624,AgainstDigitizing,(NoCP,RoadAddressCP),(NoCP,JunctionPointCP),List(Point(420026.0,7064963.0,0.0), Point(419933.0,7065099.0,0.0)),projectId,LinkStatus.Transfer,AdministrativeClass.State,FrozenLinkInterface,167.624,31341,265176,12,false,None,1537225220000L,64356,roadName,None,None,None,None,None,None)
+      )
+
+      val transferred = ProjectDeltaCalculator.partitionWithProjectLinks(allProjectLinks.filter(_.status != LinkStatus.Terminated), allProjectLinks)
```

Comment:
partitionWithProjectLinks() parametrien nimeämiset oikein päin ja korjaukset sitä kutsuviin funktioihin (kuten tähän)